requirements.txt

```diff
@@ -1,8 +1,12 @@
 fastapi==0.100.0
 fastapi-utils==0.2.1
 hjson
+numpy
+numpy-stl
 opencv-contrib-python==4.5.5.64
 pydantic~=1.10.15
+PyOpenGL==3.1.7
+PyOpenGL-accelerate==3.1.7
```

Comment:
Add instructions to download visual studio build tools.
Specify compatible Pycharm versions. (2023)

Reply:
I added instructions to `doc/setup.md` about Build Tools.

I'm not sure that there's an incompatibility between this code and a specific version of PyCharm. It is possible that we saw a bug in PyCharm, or that there is an incompatibility issue with our own computer configurations at the NRC. So the cause is unknown...